samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/SemanticChatMemorySkill.cs

```diff
@@ -34,14 +34,16 @@ public class SemanticChatMemorySkill
     /// Query relevant memories based on the query.
     /// </summary>
     /// <param name="query">Query to match.</param>
-    /// <param name="context">The SKContext</param>
+    /// <param name="chatId">Chat ID to query history from.</param>
+    /// <param name="tokenLimit">Maximum number of tokens.</param>
+    /// <param name="textMemory">Semantic text memory.</param>
     /// <returns>A string containing the relevant memories.</returns>
     [SKFunction, Description("Query chat memories")]
     public async Task<string> QueryMemoriesAsync(
         [Description("Query to match.")] string query,
```

Comment:
I don't believe you need the attributes on each parameter as the descriptions are already extracted from the triple-slash comments

Reply:
Descriptions are not extracted from comments. But you only need the [Description] on params that the AI could set (via ContextVariables). If they're being set deterministically by the Kernel (i.e. CancellationToken), then Description isn't used.

Note: ISemanticTextMemory has been removed from the SKContext and will no longer be auto-mapped to fill this arg as it once was. I recommend you pass in an ISemanticTextMemory instance in to the skill constructor instead.